test/tap/shrinkwrap-local-dependency.js

```diff
@@ -85,7 +85,7 @@ test('shrinkwrap uses resolved with file: on local deps', function (t) {
       t.comment(stderr.trim())
       t.equal(code, 0, 'npm exited normally')
       var data = fs.readFileSync(path.join(testdir, 'npm-shrinkwrap.json'), { encoding: 'utf8' })
-      t.deepEqual(
+      t.like(
```

Comment:
`deepEqual` requires exact matching of the datastructures. `like` let's you compare an incomplete datastructure against a complete one. Where they share keys they have to match.

So this change was made to allow changes to the `package-lock` format that aren't related to this test to happen w/o breaking the test.